### ISSUE TITLE
WCAG 2.2 CR: SC 2.4.11: Does required contrast between focused and unfocused states mean focused and hovered must be distinct?

### ISSUE THREAD

**Issue by jeremy-w** [w3c/wcag#2900]
The new [SC 2.4.11 "Focus Appearance"](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html) has 2 arms, and all subcriteria of either one arm or the other must be met. Both arms include the same subcriterion:

> has a contrast ratio of at least 3:1 between the same pixels in the focused and _**unfocused states**_ [emphasis added]

I'm not clear on what this means, especially with regard to hover and focus interaction.

You might also regard this as asking whether this is "focused state vs unfocused states (all of 'em)", reading it as "unfocused states, plural", or "focused state vs default state" (which can be its own can of worms to define, but is at least just one state), which would be contrasting against "the unfocused state, singular".

(I've also been trying to sort out the "do focus and hover appearances need to be distinct? is 'focused and hovered' actually a separate state that should look different from either alone? how different?" set of questions against WCAG 2.1, and I haven't come to a clear conclusion, so if you would be able to weigh in on that as of the currently active standard, that would be very helpful as well. 🙏 In that case, my main concern is whether making say a hovered button and a focused button look the same destroys focus visibility required by [SC 2.4.7](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html), since you then cannot tell which one is the keyboard focus, especially when keyboard button presses have the side effect of hiding the mouse pointer, or whether its still being visible - even if it might at times be uncertain which of the visible focuses is the keyboard one - suffices. Let me know if I should spawn a different issue for that.)

**Comment by JAWS-test** [comment]
I would say that hover does not matter. There are no requirements for that in WCAG. When comparing the focused and non-focused states, both states refer to the display without hover. A rendering with simultaneous hover state is not considered.

As a non-native speaker, I cannot say why states is used in plural.
* either the plural comes from the focused and unfocused state (because they are two different).
* or the plural should be changed to singular

**Comment by jeremy-w** [comment]
Setting hover aside, the issue comes to confounding - if there are two things that look focused, but only one is the actual keyboard focus, is focus visible (one of those is definitely focus)? Or obscured through ambiguity (I don’t know where my keyboard focus is at a glance)?

I can help with the English; there’s an ambiguity (which I’m poking at) where you can say:

- “the red thing and the blue thing”, meaning unambiguously one of each;
- “the red and blue things”, meaning two things, or maybe more, and some are red and some are blue; or
-  “the red things and the blue things”, meaning unambiguously more than one of each set.

Only one of those is ambiguous as to singular vs plural amounts of each set, and that happens to be the one in use currently.

**Comment by jeremy-w** [comment]
And I am especially asking about hover here because of how common styling hover and focus alike can be, but that can then lead to a scenario like:

- Given hover and focus look the same,
- and I’m hovering a control,
- and I tab to it,
- then there is zero change between its prior unfocused state and its new focused state,
- but there would have been had I not been hovering in.

This would be another flavor of interaction, which is why I am curious about what “unfocused state(s)”  includes.

**Comment by JAWS-test** [comment]
I suggested creating a new SC for hover: https://github.com/w3c/wcag/issues/899

However, I suspect that so far in WCAG hover effects do not play a role and thus your concerns, while valid, are not addressed in any WCAG SC.

To make this clear that hover currently plays no role in the focus SC, I suggest adding this to the Understanding: The focus indicator is only checked:
* without focus + without hover vs.
* with focus + without hover

**Comment by alastc** [comment]
> "focused and unfocused states" 

This is simply that when you are talking about focused and unfocused, that is two states, therefore plural. Hover (and other states) don't figure into it. 

In general (for 2.0-2.2) I can't think of any requirements for hover states. As the user controls the mouse, and can (for example) configure a large mouse pointer, the visibility of the hover state is not something WCAG has set a requirement for.

If that answers the query please close this issue.
